### ISSUE TITLE
593 in holdings

### ISSUE THREAD

**Issue by jenniferward** [rism-digital/muscat#1291]
Currently the 593 is repeatable in the holdings, but what do we do when someone has "Additional manuscript material"?
They will definitely have at least a printed edition (with or without annotations) so in 593$a for every copy we can expect one of the following:
- Print (the default)
- Print with autograph annotations
- Print with non-autograph annotations

$b will vary according to the material.

But when we mention the additional manuscript material, should we change the "Print" to "Additional manuscript material" (thus eliminating the "Print" designation) or should the field stay repeatable? If something is entered in the 300, how will the user know whether to connect it to the print or the manuscript? There are no material groups in holdings.

**Comment by cgueggi** [comment]
What about making the whole block "material description" repeatable? So one can add the handwritten material in the second block, incl. 593 "Additional manuscript material" and "notated music" or even a handwritten or printed libretto.

**Comment by cgueggi** [comment]
Another thougth on this topic: Since we have the possibility to describe "additional manuscript material" in the holdings, wouldn't it also make sense if we had all the fields available for the material description. That means the whole material section of the manuscript template, incl. 593 (of course), 260, 300, 590 etc. or at least some of them.

**Comment by jenniferward** [comment]
We need to find a balance because if people need such details for MSS then they should just use the MS template. But I do see an argument for 590 (why can you indicate a "score" in the 300 but if you have parts you can't list them because there is no 590?), and today a group requested the functions arr and cmp in 700$4 in holdings (which we had deleted here https://github.com/rism-digital/muscat/issues/736)—but I see their point, if they want to add names of people who heavily mark up printed editions.

**Comment by cgueggi** [comment]
I see the point. At least 590 is necessary. For the description of further details, the field "General notes" (500) can always be used.

**Comment by ahankinson** [comment]
`arr` and `cmp` would be on the bibliographic record. They have a separate role if they're marking up an arranged or composed piece, even if they were also the arranger or composer -- likely `ann` (Annotator), `cmm` (Commentator), `crr` (Corrector), or `pfr` (Proofreader).

**Comment by ahankinson** [comment]
The other possibility is that we change the 593 "Print with **autograph** annotations" to "Print with **manuscript** annotations". Then we know that it's a) a print, and b) it has manuscript annotations. Then we can use 700$4 to describe who did the annotations. And we wouldn't need a repeatable 593 on holdings, since it covers both the print and the manuscript parts.

**Comment by cgueggi** [comment]
I think that's a tricky topic. When a print is arranged in a way that it is nearly a new work (and so a "arr" is justified) imo a separate Muscat entry should be created (if necessary the print entry still can remain). The repeatable "material description" in holdings is meant (again in my opinion) to describe further handwritten material additional to the printed material such as copied or added parts. That could be when the circumstances of an audition differed from the original composition, f. e. the orchestra has 2 cl instead of 2 ob and thus on transposed the ob parts to cl parts. But that does not mean an "arrangement" in its proper sense. Sometimes we have just copied handwritten parts, f.e. of violins, because there were more than two vl 1 players in a specific concert.
Further, I think the "annotations" on a print are not meant in the sense of an arrangement but more in the sense of added comments, annotations,  small corrections etc. by the composer or another person, which is quite rare but belong to the holding and not to the bibliographic record. Therefore it would be maybe useful to give all possible funcitions in 700$4 to choose from also in the holdings.

**Comment by ahankinson** [comment]
> The repeatable "material description" in holdings is meant (again in my opinion) to describe further handwritten material additional to the printed material such as copied or added parts.

The holdings, in all our systems (Muscat, RISM Online, OPAC) are much more "hidden" than the source records -- users cannot retrieve holdings records directly, they must always go through the source records. So providing extensive modelling in holdings records is effectively hidden from a search and retrieval perspective, since we cannot provide users with the tools to search or find this data directly. 

In RISM Online, for example: If we have content in the holdings, our approach is to "merge" this data in the source records, since users can only retrieve sources. But then the data in the holding records is indistinguishable from the data that is in the source record, unless we add an ever-more expanding set of facets / search fields / etc. that target only those specific bits of the record. If we add more complexity to our records (by, for example, providing repeatable material groups) then we need to provide siginficantly more complex retrieval mechanisms for them to find it again, and these search options target increasingly smaller portions of our entire dataset.

So to me the question is: Does adding complexity to the holdings records, in order to more accurately describe a single holding, justify the necessary complexity in our users' search experience to find these holdings again? If we make a really complex way of "hiding" things through increasingly detailed use of data across several fields, will this actually be detrimental to our users ever be able to "find" them again? (**)

Or to rephrase it: Is a single non-repeatable field actually more descriptive, since it will remove variability and provide more consistent data across all cataloguers and records, and therefore allow us to provide more accurate retrieval mechanisms for our users to find what they are looking for. 

What I see as our biggest challenge at the moment is data consistency -- too many records structured by too many people in too many different and "creative" ways, such that it's very difficult to do any one search that finds all the relevant things you might be interested in...


(**) My cataloguing teacher in my library degree described libraries as institutions dedicated to hiding books, and that cataloguers' jobs was to provide the means for people to find them again, and not to be overly clever in how they do it. ("It doesn't matter if you're right, it only matters that you're consistent." was one of his lines) This stuck with me, and has meant that given the choice between more technically accurate but complex ways of describing a book, and simpler but more straightforward ways of actually finding it in the systems we have, I will generally favour the latter.

**Comment by cgueggi** [comment]
I think we see the issue from very different angles. You from a technical perspective, me from the musical source. As a cataloguer, I want to describe the existing source as accurately as possible. This means that I also give users detailed descriptions of additional materials so that they can get a more accurate picture of them. In the end, this is precisely the information that differs from other holdings, which perhaps makes the one specific source more interesting for a user. In the case of manuscripts, we also have the option of duplicating the material and, for example, indicating a whole layer of material with a different scribe.

I see the problem with searchability, but I think that in these cases in particular it is not necessary for it to be discoverable through a search query. In my view, these additions are in fact simply intended as additional information. Because ultimately the user usually searches for the print itself and then, if necessary, sees the additional information that he may or may not find interesting.

Of course, I cannot estimate what consequences this will have for the technical development of both Muscat and rism.online. Perhaps I am simply too analogue (reactionary) in this question ;-). But as I mentioned earlier, my aim is to give as accurate a picture of a source as possible, so that the user can get an equally accurate idea of it. This applies to manuscripts as well as to holdings of printed sources.

Perhaps we need to discuss the issue face-to-face in more detail, possibly together with @lpugin and @xhero.

**Comment by xhero** [comment]
What should we do here?

**Comment by cgueggi** [comment]
Maybe this needs further discussions. In the meantime, I worked with the general note field (500) to describe the additional, handwritten material. Let's postbone this topic to a later updat, ok?

**Comment by xhero** [comment]
Is this still relevant with the new 593?

**Comment by cgueggi** [comment]
From the cataloguer's view I can live with it the way things are now. Recently, however, I have no longer given details in 300 for additional handwritten material, but simply named the existing parts in 500. See for example: https://muscat.rism.info/admin/sources/990068732#holding_264384
I think it's just important that the guidelines are clear in this point.

**Comment by jenniferward** [comment]
I also have started putting things in 500 in holdings because otherwise the 300 can get wild. Is it still out of the question to add a 590 in holdings to list parts? 
But generally if @cgueggi is a happy cataloger then I'm happy.

**Comment by xhero** [comment]
@cgueggi are you a happy cataloger? :) Is this ticket still relevant?

**Comment by cgueggi** [comment]
Yes, I'm happy with that. Just make clear in the guidelines how one has to proceed in such cases. Maybe a short addition would be helpful (f.e. "Further descriptions on the manuscript material can be given in 500").
